examples/mobile/UIComponents/components/popup/index.html

```diff
@@ -13,44 +13,19 @@
 	<div class="ui-page" id="popup-demo-page">
 		<div class="ui-header" data-position="fixed">
 			<h1>
-				Popup
+				Dialogs
```

Comment:
Folder name is the 'popup' but title and ux guide are 'Dialogs'.
In wearable, it seems to be used as popup :(
How about change the folder name also?

Reply:
Sure. I'll change the folder name popup-> Dialog according to UX guide.